composer.json

```diff
@@ -6,8 +6,8 @@
     "license": "MIT",
     "require": {
         "php": "^7.0",
-        "tarantool/client": "v0.2.0",
-        "rybakit/msgpack": "v0.2.1"
+        "tarantool/client": "^0.2.0",
+        "rybakit/msgpack": "^0.2.1"
```

Comment:
I would also move msgpack to "suggest", as there is no hard dependency on this package.